README.md

```diff
@@ -208,6 +211,29 @@ available [here](http://commons.apache.org/proper/commons-jexl/reference/syntax.
   set column hrlywage Math:abs(toDouble(hrlywage))
 ```
 
+## Quantize
+This directive quantizes a continous value of a column through a range table
+specified. The quantization ranges are all real numbers, with low specifying the low end of the
+ range and high specifying the high end of the range. Associated with the range is the
+ value that if the incoming value falls in the range it would be assigned that value.
+ The range is a closed range - [low:high] = {x | low <= x <= high}. Also, the high endpoint
+ should be greater than low endpoint.
+
+**Specification**
+```
+  quantize {source-column} {destination-column} {quantization-table}
+```
+
+* source-column : Name of the column which has to be quantized
+* destination-column : Name of the column to which the quantized value should be added.
+* quantization-table : Specifies the quantization table in the following format low:high=value[,low:high=value]*
```

Comment:
What does this mean `value[,low:high=value]*` ? Does this depicts that the user can have multiple ranges?